flycheck-haskell.el

```diff
@@ -143,6 +143,9 @@ Return the configuration."
   (or (flycheck-haskell-get-cached-configuration cabal-file)
       (flycheck-haskell-read-and-cache-configuration cabal-file)))
 
+(defconst flycheck-haskell-cabal-config "cabal.config"
```

Comment:
I'd cross that bridge when it comes up, especially since I don't think it's coming as soon as we'd like.